### PR TITLE
Ft clinic Database

### PR DESCRIPTION
**Pull request by zbahati**
## In this milestone we did the following:
- add database
- add patients table
- add medical_histories table
- add invoices table
- Add constraints for patients
- Add foreign index of patients
- Add foreign index for medical histories,
- Create table invoice_items,
- Creates table treatments,
- Add foreign key for invoices,
- Add foreign key treatments,
-Add foreign key index
- Create Join table medi_treat_relation,
- Add foreign key for medical histories and treatments,
- Modify readme,

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+-- JOIN TABLE FOR medical histories and Treatements table
+CREATE TABLE medic_treat_relation(
+  treatment_id INT,
+  medical_histories_id INT
+);
+
+ -- FOREIGN KEY FOR medecal_histories and treatments
+ALTER TABLE medic_treat_relation ADD CONSTRAINT treatment_fk FOREIGN KEY(treatment_id) REFERENCES treatments(id);
+ALTER TABLE medic_treat_relation ADD CONSTRAINT medical_histories_fk FOREIGN KEY(medical_histories_id) REFERENCES medical_histories(id);
```
Comment (muneebulrehman) [inline schema_based_on_diagram.sql:75]: Both of these foreign keys should have indexes created on them. Kindly do so as you did for the rest.

File: README.md [hunk 2]
```diff
@@ -1 +1,285 @@
-# clinic_db
\ No newline at end of file
+# curriculum-databases-projects-template
+
+> This template should be used for database related projects at Microverse.
+> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+
+
```
Comment (muneebulrehman) [inline README.md]: - Optional. I would recommend removing this part form your README file to keep it clean.

File: README.md [hunk 3]
```diff
@@ -1 +1,285 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [COMING SOON](<replace-with-your-deployment-URL>)
```
Comment (muneebulrehman) [inline README.md]: I would recommend removing this section from your README file to keep it clean since it is not relevant to this project.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR
- Contribution by both partners
- Good job creating a database from the provided diagram.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @zbahati, and Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**